integration-test/common/src/test/java/com/tencent/rss/test/IntegrationTestBase.java

```diff
@@ -96,9 +96,6 @@ protected static ShuffleServerConf getShuffleServerConf() {
 
   protected static void createCoordinatorServer(CoordinatorConf coordinatorConf) throws Exception {
     coordinators.add(new CoordinatorServer(coordinatorConf));
-    coordinatorConf.setInteger("rss.rpc.server.port", COORDINATOR_PORT_2);
-    coordinatorConf.setInteger("rss.jetty.http.port", JETTY_PORT_2);
```

Comment:
we should need two coordinators

Reply:
@jerqi yes, we should add case to test multiple coordinator.

Reply:
There are so many errors in ut's log that fail to send heartheat to secondary coordinator. @colinmjj @duanmeng If you don't have strong opinions on this, i will revert this patch.

Reply:
@jerqi I think it's UT's problem, and ut should be fixed.
We shouldn't enable 2 coordinators with all situations.

Reply:
OK, Let's remove extra client.